src/games/strategy/engine/framework/startup/launcher/LocalLauncher.java

```diff
@@ -49,20 +49,8 @@ protected void launchInNewThread(final Component parent) {
         game.setRandomSource(new ScriptedRandomSource());
       }
       m_gameData.getGameLoader().startGame(game, gamePlayers, m_headless);
-    } catch (final IllegalStateException e) {
+    } catch( IllegalStateException e ) {
       exceptionLoadingGame = e;
-      Throwable error = e;
-      while (error.getMessage() == null) {
-        error = error.getCause();
-      }
-      final String message = error.getMessage();
-      m_gameLoadingWindow.doneWait();
-      SwingUtilities.invokeLater(new Runnable() {
-        @Override
-        public void run() {
-          JOptionPane.showMessageDialog(null, message, "Warning", JOptionPane.WARNING_MESSAGE);
-        }
-      });
```

Comment:
Also notice that 'Exception' and 'IllegalState' are handled the same way now. It seems likely the same really random error would hit either the IllegalState block, or Exception, who knows what the code would do, and so it doesn't seem to reason why they should be handled differently.

In the previous case it was obvious the IllegalStateException was hijacked for the map not found purpose, for example note the extremely generous detail on the error message. We do not have examples where an IllegalState is so well documented with the messaging alone.